test/ProjectOrigin.Electricity.Example/WithoutWalletFlow.cs

```diff
@@ -99,9 +99,37 @@ public async Task<int> Run()
         Console.WriteLine($"Claiming Consumption Granular Certificate");
         await SendClaim(ownerKey, consClient, consCertId, allocationId);
 
+        // ------------------  withdraw production ------------------
+        Console.WriteLine($"Withdraw Production Certificate");
+        await SendWithdraw(ownerKey, prodClient, prodCertId);
+
+        // ------------------  unclaim consumption ------------------
+        Console.WriteLine($"Unclaim Consumption Certificate");
+        await SendUnclaim(allocationId, ownerKey, consClient, consCertId);
+
         return 0;
     }
 
+    private static async Task SendUnclaim(Guid allocationId, IHDPrivateKey ownerKey, RegistryService.RegistryServiceClient prodClient, FederatedStreamId prodCertId)
+    {
+        var withdrawnEvent = ProtoEventBuilder.CreateUnclaimedEvent(allocationId);
+
+        // Sign the event as a transaction
+        var signedTransaction = ownerKey.SignTransaction(prodCertId, withdrawnEvent);
+
+        // Send transaction to registry, and wait for committed state
+        await prodClient.SendTransactionAndWait(signedTransaction);
+    }
+
+    private static async Task SendWithdraw(IHDPrivateKey ownerKey, RegistryService.RegistryServiceClient prodClient, FederatedStreamId prodCertId)
+    {
+        var withdrawnEvent = ProtoEventBuilder.CreateWithdrawnEvent();
+
+        var signedTransaction = ownerKey.SignTransaction(prodCertId, withdrawnEvent);
+
+        await prodClient.SendTransactionAndWait(signedTransaction);
+    }
+
```

Comment:
in the two methods, the RegistryServiceClient, should just be called client and not prodClient